### DISCUSSION TITLE
Suggest new services to embed.

### DISCUSSION THREAD

**Discussion by insertish** [stoatchat/discussions#29] [Feature Suggestions]
January currently supports:
- YouTube
- Twitch
- Spotify
- Soundcloud
- Bandcamp

Suggest new services to support below.

Edit: currently closing discussion and moving into issues.

**Comment by div2005** [comment]
Also we need this:

- Giphy
- Tenor
- Imgur

**Reply by insertish** [reply]
Using https://github.com/revoltchat/january/issues/20 to track this.

**Comment by insertish** [comment]
Need to add support for Revolt invites and autumn content

**Comment by gdude2002** [comment]
It'd be nice to support a little embed for shortened URLs and redirects that shows what the target URL would be

**Reply by insertish** [reply]
Using https://github.com/revoltchat/january/issues/21 to track this.

**Comment by Andre601** [comment]
maybe [gfycat](https://gfycat.com/)?

**Reply by insertish** [reply]
Using Using https://github.com/revoltchat/january/issues/20 to track this.

**Comment by Rexogamer** [comment]
i'm aware this is planned but twitter
